cmd/plural/cd_stacks.go

```diff
@@ -0,0 +1,96 @@
+package plural
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/urfave/cli"
+
+	"github.com/pluralsh/plural-cli/pkg/config"
+	"github.com/pluralsh/plural-cli/pkg/stacks"
+	"github.com/pluralsh/plural-cli/pkg/utils/git"
+)
+
+func (p *Plural) cdStacks() cli.Command {
+	return cli.Command{
+		Name:        "stacks",
+		Subcommands: p.cdStacksCommands(),
+		Usage:       "manage CD stacks",
+	}
+}
+
+func (p *Plural) cdStacksCommands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:   "gen-backend",
+			Action: latestVersion(p.handleGenerateBackend),
+			Usage:  "generate '_override.tf' to configure a custom terraform backend",
+			Flags: []cli.Flag{
+				cli.StringFlag{Name: "address", Usage: "terraform backend address", Required: false},
+				cli.StringFlag{Name: "lock-address", Usage: "terraform backend lock address", Required: false},
+				cli.StringFlag{Name: "unlock-address", Usage: "terraform backend unlock address", Required: false},
+			},
+		},
+	}
+}
+
+var backendSurvey = []*survey.Question{
+	{
+		Name:     "address",
```

Comment:
We shouldn't ask for these addresses, we should instead generate the appropriate ones from a given service id or `@{handle}/name` arg.